src/AppBundle/Twig/AppExtension.php

```diff
@@ -49,9 +49,9 @@ public function markdownToHtml($content)
     }
 
     // the name of the Twig extension must be unique in the application. Consider
-    // using 'app.extension' if you only have one Twig extension in your application.
+    // using 'app_extension' if you only have one Twig extension in your application.
```

Comment:
form type names and twig extension names cannot conflict. They are identifiers in different systems. We will never register a form type in Twig or a twig extension in the form registry (it is impossible)